### ISSUE TITLE
kill jobspec shoud be pkill

### ISSUE THREAD

**Issue by dschep** [dthree/cash#7]
GNU coreutils (whis what this emulates, not Linux) `kill` only kills process by pid. `pkill` is the POSIX command for killing processes by name (which GNU&BSD coreutils implement).

**Comment by dthree** [comment]
That's true. I kind of guiltily added that in there, but after I implement `pkill` I'll remove the feature.
